### ISSUE TITLE
Trouble installing as template

### ISSUE THREAD

**Issue by TBleader** [ivoyager/ivoyager#8]
**What's the problem?**
I'm having trouble installing this as a project template in godot. I followed the instructions [here](https://ivoyager.dev/forum/index.php?p=/discussion/54/downloads-git-instructions-how-to-make-a-new-project-from-the-project-template#latest)

I've used gitkraken and it imported as expected. I've got the following directory structure

> ivoyager]$ ls
> 3RD_PARTY.txt       jupiter-icon-128.png         project.godot
> CREDITS.md          jupiter-icon-128.png.import  README.md
> export_presets.cfg  LICENSE.txt                  replace_me
> ivoyager            pale_blue_dot.png            universe.gd
> ivoyager_assets     pale_blue_dot.png.import     universe.tscn
> 

ivoyager contains the content on this branch. assets were downloaded from the 0.0.7 as instructed. When importing into the godot editor I encounter these issues:

> ERROR: load_interactive: Cannot open file 'res://ivoyager/data/text/entities_text.en.translation'.
>    At: core/io/resource_format_binary.cpp:999.
> ERROR: _load: Failed loading resource: res://ivoyager/data/text/entities_text.en.translation. Make sure resources have been imported by opening the project in the editor at least once.
>    At: core/io/resource_loader.cpp:278.
> ERROR: load_interactive: Cannot open file 'res://ivoyager/data/text/gui_text.en.translation'.
>    At: core/io/resource_format_binary.cpp:999.
> ERROR: _load: Failed loading resource: res://ivoyager/data/text/gui_text.en.translation. Make sure resources have been imported by opening the project in the editor at least once.
>    At: core/io/resource_loader.cpp:278.
> ERROR: load_interactive: Cannot open file 'res://ivoyager/data/text/long_text.en.translation'.
>    At: core/io/resource_format_binary.cpp:999.
> ERROR: _load: Failed loading resource: res://ivoyager/data/text/long_text.en.translation. Make sure resources have been imported by opening the project in the editor at least once.
>    At: core/io/resource_loader.cpp:278.
> ERROR: get_dependencies: Cannot open file 'res://.import/Bennu.2000_1_1.glb-0032803fc715a155e9ded522a7738b8e.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Deimos.4000_1_1000.glb-f43d9c2a646ddd3f1bc0341dff71f497.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Eros.1500_1_100.glb-c4e30118bc81cb8e2c0e0f001cfe20f4.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Hyperion.4000_1_1000.glb-9093abc38f94147a25d4742c8b63cb0e.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Itokawa.2500_1_1.glb-1541d94a082ade7bfc2a8887a5404a83.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Phobos.4000_1_1000.glb-4177d00b6d1785855bf1adefbff6e82e.scn'.
>    At: core/io/resource_format_binary.cpp:1048.
> ERROR: get_dependencies: Cannot open file 'res://.import/Vesta.4000_1_1000.glb-52a0bd17527658b5f3d07a5fced41008.scn'.
>    At: core/io/resource_format_binary.cpp:1048.

The editor then hangs and consumes 100% CPU despite never accomplishing anything else, even after letting it sit for 30 minutes. I'm scratching my head at this one, these files don't seem to exist anywhere in the repos from what I could tell so I'm not sure why it's trying to grab them? Any help is much appreciated. Sorry if this is the wrong place to post, I tried making two accounts on the forums but it never sent any confirmation emails for either despite trying to resend it multiple times for both... Separate issue that is haha. 

On linux 64 bit, using godot linux 64 bit editor (not the mono one, not sure if that is needed since it seemed this project was gdscript and c++?)

**Comment by charliewhitfield** [comment]
FYI, I'll be uploading v0.0.9 today or tomorrow. It has fixes to work with the new Godot 3.3 (which just dropped last week) and [many other improvements](https://github.com/ivoyager/changelogs). So please try that in a day or so if we don't resolve the current problem (or even if we do!).

So, first thing, can you check your ivoyager and Godot versions? Our last "release" v0.0.8 was developed using Godot 3.2.3 and has compatibility problems with Godot 3.3. (If you are using current _master_ branch code, it should work with Godot 3.3 although there is a manual fix you have to make to assets as described in 1st line of [changelog](https://github.com/ivoyager/changelogs#readme).)

When you first open the project, Godot has to import all of the graphics (these are in ivoyager_assets that you downloaded separately). Godot does this on the CPU, and it can take a while depending on your system (for me, maybe 5 min on my desktop to 30+ min on my old laptop; both Windows 10 systems). The wall-of-error output seems to be a feature during this process. The way to see if something is happening is to open .import folder, sort by date, and watch whether new content is being generated. My old laptop sometimes hangs forever on the largest image file, which is one of the 2 starmap images (you might try removing starmap_16k and keeping only starmap_8k). Generally, I quit and restart Godot Editor after import. I never see any of these errors after restarting. (If you do get past import problem, be sure to verify Phobos, Deimos & Hyperion models are really there.)

There were changes to Godot import system in 3.3. I haven't looked for a difference but perhaps 3.3 will speed up the import and clean up some errors messages.

Sorry about the forum signup problem. ~~It may be that our email got blacklisted by your mail server. Can you try emailing us at address listed on our Github front page? (Top of [https://github.com/ivoyager](https://github.com/ivoyager).) We can try to troubleshoot from there.~~ EDIT: I fixed some no-longer-functioning SMTP mail settings on our forum. Try logging in (to existing user you set up) and hit resend email verification (it should still be there in lower-left). It should work this time.

**Comment by TBleader** [comment]
Ah so my issue might've been two-fold. I was using Godot 3.2.3 with the v0.0.9-dev build (not sure if that specifically caused any extra issues) and I also vastly underestimated how long that import takes... As it used 100% of my CPU making doing other things while it runs in the background kinda painful I set it overnight and it took 3 hours. Not sure why it's so slow compared to what you are seeing. I've got an i7-8700K @ 4.6GHz so it isn't like an awful CPU. It did seem to stick to one core, maybe even just one thread even so I wonder if that was a bottleneck. In the morning I quit and restarted and can confirm no errors, and the assets it claimed were missing are there. Hopefully I won't have to do asset imports often :laughing: 

Yup was able to get the email, I can direct future non-github appropriate questions/issues there now. Thanks for your help :)
